lib/workers/repository/update/pr/index.ts

```diff
@@ -82,16 +85,33 @@ export async function ensurePr(
   );
 
   const config: BranchConfig = { ...prConfig };
-
+  const prFingerprint = fingerprint(config);
   logger.trace({ config }, 'ensurePr');
   // If there is a group, it will use the config of the first upgrade in the array
   const { branchName, ignoreTests, prTitle = '', upgrades } = config;
   const dependencyDashboardCheck =
     config.dependencyDashboardChecks?.[config.branchName];
-  // Check if existing PR exists
+  // Check if PR already exists
   const existingPr = await platform.getBranchPr(branchName);
   if (existingPr) {
     logger.debug('Found existing PR');
+    const prCache = getPrCache(branchName);
+    if (prCache) {
+      logger.debug({ prCache }, 'Found existing Pr cache');
+      const lastEditTime = prCache.lastEdited;
+      // return if pr cache is valid and pr was last edited before a day
+      if (
+        prFingerprint === prCache.fingerprint &&
+        getElapsedHours(lastEditTime) > 24
+      ) {
+        logger.debug('Cache is valid, skipping PR update');
+        return { type: 'with-pr', pr: existingPr };
+      }
+      logger.debug('Invaild cache, processing PR');
+    } else {
+      logger.debug('Pr cache not found, creating new');
```

Comment:
```suggestion
      logger.debug('PR cache not found, creating new');
```

Will this log even when repo caching is not enabled? We should not be noisy or alarm users if repo cache is disabled. In that case we should log nothing at all.